gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-bin.zip
```

Comment:
We need this downgrade to avoid the issue related to Gradle 8.10.

https://youtrack.jetbrains.com/issue/KT-67162/KGP-Kotlin-Native-with-Isolated-Projects-kotlinNativeBundleBuildService-cannot-be-changed-any-futher